jablib/src/main/java/org/jabref/logic/importer/fetcher/citation/semanticscholar/SemanticScholarCitationFetcher.java

```diff
@@ -33,6 +34,12 @@ public String getAPIUrl(String entry_point, BibEntry entry) {
                 + "&limit=1000";
     }
 
+    public String getAPIUrl(BibEntry entry) {
```

Comment:
Rename to `getUrlForCitationCount` - and remove fields `title,authors,year` and all others but `citationCount`

Because: The other thing already uses the full information - we just need the citaton count!!!